### DISCUSSION TITLE
GraphQL API Documentation

### DISCUSSION THREAD

**Discussion by SuzanaStevic** [requarks/wiki#5138] [General]
Does Wiki have an option to automatically generate GraphQL API(s) documentation? I am searching for a tool that can help me document GraphQL API for several projects since we need to provide the customer not only API schema but some type of static document which will include more details and which will be more clear. Most of the "automated" documents are for REST APIs, not GraphQL. Thank you.

**Comment by vuducmanh11** [comment]
You can find graphql doc in grapql playground. Access url http://[wikilink]/graphql
![image](https://user-images.githubusercontent.com/22274112/170173207-09d22377-549c-4f4e-95ae-03af1e3a5b22.png)

**Reply by SuzanaStevic** [reply]
@vuducmanh11  Thank you very much, but we already use the playground, the idea is to find some tool that will help us to add more details to the API doc, more descriptions, examples, etc so that everything is in one place, but it seems that for we will have to do it manually.... :)

**Reply by NGPixel** [reply]
This is planned for v3. We'll have detailed examples and more.

**Reply by SuzanaStevic** [reply]
@NGPixel  Thanks for the info. :) If it can be helpful, in our case we need to "create" GraphQL API static doc similar to REST API. We need to add as many descriptions as possible for queries and as well other details. Perhaps it would be great if we can add those descriptions directly to the playground (some type of editor) and export a static doc similar to one that can be exported/published via Postman and other API tools....Or to have the option to run a few queries and save them as examples, similar to Banana Cake Pop but with some type of export/publish/edit option so that we can add info if needed, or remove if not... Thanks again, you made my day.
